src/add_display_name.rs

```diff
@@ -18,42 +18,72 @@ impl Component {
     }
 }
 
-pub struct AddDisplayNameVisitor;
+impl Component {
+    pub fn create_display_name_stmt(&self) -> ModuleItem {
+        ModuleItem::Stmt(
+            Stmt::Expr(ExprStmt {
+                span: DUMMY_SP,
+                expr: Box::new(Expr::Assign(AssignExpr {
+                    span: DUMMY_SP,
+                    op: AssignOp::Assign,
+                    left: PatOrExpr::Expr(Box::new(Expr::Member(MemberExpr {
+                        span: DUMMY_SP,
+                        obj: Box::new(Expr::Ident(Ident::new(self.name.clone(), Span { ctxt: self.ctx, ..DUMMY_SP }))),
+                        prop: MemberProp::Ident(Ident::new(JsWord::from("displayName").into(), DUMMY_SP))
+                    }))),
+                    right: Box::new(Expr::Lit(Lit::Str(Str::from(self.name.clone()))))
+                }))
+            })
+        )
+    }
+}
+
+pub struct AddDisplayNameVisitor {
```

Comment:
Seems like we can safely derive it?

```suggestion
#[derive(Default)]
pub struct AddDisplayNameVisitor {
```

Reply:
Thank you! I'll update the code.